forms-web-app/src/validators/appellant-submission/application-number.js

```diff
@@ -1,7 +1,12 @@
 const { body } = require('express-validator');
 
 const rulePlanningApplicationNumber = () =>
-  body('application-number').notEmpty().withMessage('Enter your planning application number');
+  body('application-number')
+    .notEmpty()
+    .withMessage('Enter your planning application number')
+    .bail()
+    .isLength({ min: 1, max: 30 })
+    .withMessage('Application numbers cannot have more than 30 characters');
```

Comment:
To match https://github.com/foundry4/appeal-planning-decision/pull/152#discussion_r533127584